manifest.json

```
@@ -2,7 +2,7 @@
   "id": "calendar",
   "name": "Calendar",
   "description": "Calendar view of your daily notes",
-  "version": "1.5.10",
+  "version": "1.5.11",
```

Could you remove these changes for now? I'll bump the version when I'm ready to cut the next release.

```suggestion
  "version": "1.5.10",
```

---

thank you, fixed
also fixed for another PR https://github.com/liamcain/obsidian-periodic-notes/pull/53